addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3122,6 +3146,55 @@ const SnippetOptionWidget = Widget.extend({
 
         _restoreTransitions();
     },
+    /**
+     * Redirects background calls to a specific CSS property.
+     *
+     * @param {boolean} previewMode - @see this.selectClass
+     * @param {string} widgetValue
+     * @param {Object} params
+     * @returns {Promise|undefined}
```

Comment:
As soon as your method is async, it cannot return "undefined" (which is good, just update the doc)